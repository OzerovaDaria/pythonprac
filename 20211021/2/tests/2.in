:cos x cos(x)
cos 1
quit all
