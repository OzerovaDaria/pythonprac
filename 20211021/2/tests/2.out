6
1000
0.6483608274590867
fun:4strokaz:7
